docs/source/access/accessing_gedi_l4a_data.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7872f012",
+   "metadata": {},
+   "source": [
+    "## Accessing Global Ecosystem Dynamics Investigation (GEDI) Level 4A Granule Data\n",
+    "\n",
+    "In this example, we demonstrate how to access GEDI Level 4A granule data on the MAAP ADE.\n",
+    "\n",
+    "Within your Jupyter Notebook, start by importing the `maap` package. Then invoke the MAAP, setting the `maap_host` argument to `api.ops.maap-project.org`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c623a405",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import the maap package to handle queries\n",
+    "from maap.maap import MAAP\n",
+    "# invoke the MAAP using the MAAP host argument\n",
+    "maap = MAAP(maap_host='api.ops.maap-project.org')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b19badfc",
+   "metadata": {},
+   "source": [
+    "Search for a granule using the `searchGranule` function (for more information on searching for granules, see [Searching for Granules in MAAP](https://docs.maap-project.org/en/latest/search/granules.html)). Note that we can use `searchGranule`'s `cmr_host` argument to specify `cmr.maap-project.org` as the CMR instance. Then utilize the `getData` function, which downloads granule data if it doesn't already exist locally. We can use `getData` to download the first result from our granule search into the file system and assign it to a variable (in this case `download`)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "fdcabae0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# search for granule data using CMR host name, collection concept ID, and Granule UR arguments\n",
+    "results = maap.searchGranule(\n",
+    "    cmr_host='cmr.maap-project.org',\n",
+    "    collection_concept_id='C1202028193-NASA_MAAP',\n",
+    "    granule_ur='GEDI_L4A_AGB_Density_V2_1.GEDI04_A_2019107224731_O01958_01_T02638_02_002_02_V002.h5')\n",
+    "# download first result\n",
+    "download = results[0].getData()"
```

Comment:
Hmmm. If downloading to a bucket, I would suggest `/projects/my-private-bucket`, but @wildintellect, would it be better to download to a non-bucket location in the example? Just thinking that we might not want to get people in the habit of putting to a bucket by default. Perhaps download locally, and the copy only what's necessary to a bucket afterwards?

Reply:
I would download to a local folder not S3 backed, `/projects/local_data/` - user would need to make this folder first.
This highlights a bigger gap in the docs for later, we don't really give advice on how to manage files.

Reply:
@wildintellect, to wrap up this PR and avoid potential confusion, how do you propose we word this? Should we avoid the `local_dir` variable and call `getData` without an argument, and simply indicate that the granule file will be downloaded to the current directory? Something else?

Reply:
I think calling it with an arg is a good idea, otherwise users will never know it's even possible. The path we pick should not be in any folder with `-buckets`. My suggestion above stands, however the directory needs to exist first. So add a mkdir if not exists before doing the download.